### PR TITLE
Add install requirement for Dotnet

### PR DESCRIPTION
**Pull request by ilanbiala**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,7 @@ clean:
 	dotnet clean
 
 install:
+	@dotnet --version || (echo "Dotnet is not installed, please install Dotnet CLI"; exit 1);
```
Comment (efossier) [inline Makefile]: did you test this on windows?

Comment (efossier) [inline Makefile]: not sure if the command is also `dotnet`

Comment (ilanbiala) [inline Makefile]: I didn't test this on Windows, but it seems like it's the same: https://www.microsoft.com/net/core#windowscmd

Comment (efossier) [inline Makefile]: ok

**Pull request merged**